src/Frontend/Modules/Location/Layout/Templates/Index.html.twig

```diff
@@ -24,12 +24,18 @@
             </div>
 
             <div itemprop="name" class="hidden"><strong>{{ locationItem.title }}</strong></div>
+            {% if locationItem.add_map_link_to_address %}
```

Comment:
not a fan of this implementation, this will make it harder to copy the text of the address,

a link underneath the address would seem more logical to me